scripts/build-backend.sh

```diff
@@ -3,3 +3,8 @@ CURR=$(dirname "${BASH_SOURCE[0]}")
 
 docker build -t ${REGISTRY}webrecorder/browsertrix-backend:latest $CURR/../backend/
 
+if [ -n "$REGISTRY" ]; then
+    TAG=`docker images | grep ${REGISTRY}webrecorder/browsertrix-backend | awk '{ print $3 }'`
+    docker tag $TAG ${REGISTRY}webrecorder/browsertrix-backend:latest
+    docker push ${REGISTRY}webrecorder/browsertrix-backend
```

Comment:
Is the `docker tag` part needed here, since we're building the image?
It seems just running:
```
docker build -t localhost:32000/webrecorder/browsertrix-backend:latest
docker push localhost:32000/webrecorder/browsertrix-backend:latest
```
should be sufficient? 
At least that's what it seems from the docs at: https://microk8s.io/docs/registry-built-in - they could be more clear though

Reply:
Right. It doesn't necessary. Will update it!